tslint.json

```diff
@@ -0,0 +1,100 @@
+{
+    "defaultSeverity": "error",
+    "extends": "typestrict",
+    "jsRules": {},
+    "rules": {
+        "arrow-parens": true,
+        "ban-ts-ignore": true,
+        "import-spacing": true,
+        "indent": {
+            "options": [
+                "spaces",
+                2
+            ]
+        },
+        "max-line-length": {
+            "options": {
+                "limit": 120
+            }
+        },
+        "no-debugger": true,
+        "no-console": true,
+        "new-parens": true,
+        "only-arrow-functions": {
+            "options": [
+                "allow-declarations",
+                "allow-named-functions"
+            ]
+        },
+        "no-consecutive-blank-lines": [
+            true,
+            2
+        ],
+        "no-irregular-whitespace": true,
+        "no-trailing-whitespace": {
+            "options": [
+                "ignore-template-strings"
+            ]
+        },
+        "no-unused-variable": false, // disabled because deprecated
+        "number-literal-format": true,
+        "quotemark": {
+            "options": [
+                "single",
+                "avoid-template",
+                "avoid-escape"
+            ]
+        },
+        "semicolon": [
+            true,
+            "always"
+        ],
+        "space-before-function-paren": {
+            "options": {
+                "anonymous": "always",
+                "asyncArrow": "always",
+                "constructor": "never",
+                "method": "never",
+                "named": "never"
+            }
+        },
+        "trailing-comma": {
+            "options": {
+                "multiline": "always",
```

Comment:
This just makes reading diffs easier since when adding a new property to an object literal/element to an array literal, etc doesn't require adding a comma to the line above and causing extra change noise in commits.